launcher/ui/pages/instance/InstanceSettingsPage.ui

```diff
@@ -595,6 +595,48 @@
          </layout>
         </widget>
        </item>
+       <item>
+        <widget class="QGroupBox" name="accountGroupBox">
+         <property name="title">
+          <string>Set a account to use</string>
```

Comment:
an*

Reply:
English is not my native. I'm too lazy to check grammar.